packages/inventory/src/components/table/InventoryList.scss

```diff
@@ -84,7 +84,7 @@
         }
 
         * {
-            font-size: var(	--pf-global--FontSize--lg);
+            font-size: var(--pf-global--FontSize--md);
```

Comment:
I'd rather achieve this in app bundle than setting this globally for all apps.
```CSS
table.ins-c-entity-table.pf-c-table .ins-composed-col * {
  font-size: var(--pf-global--FontSize--md);
}
```

This in your SCSS file should do the trick.

Reply:
Ah yes that makes a lot more sense to do it that way. I'll go ahead and close this out then 👍